packages/scripts/src/framework/version-bump.sh

```diff
@@ -1,49 +1,73 @@
-#!/bin/bash
+#!/bin/bash -e
+
+LATEST_VERSION=$(npm view @chainlink/external-adapter-framework version)''
 
 function bump() (
-  cd $1
-  echo $1
-  
-  # Get current version
-  current_version=$(jq -r '.dependencies."@chainlink/external-adapter-framework"' package.json)
-  # Install latest version, pinned
-  yarn add @chainlink/external-adapter-framework -E > /dev/null # Silence output
-  latest_version=$(jq -r '.dependencies."@chainlink/external-adapter-framework"' package.json) # Get final version
-
-  if [[ $current_version == $latest_version ]] ; then
-    echo "Package $1 already at latest version $latest_version"
-  else
-    echo "Bumped framework version in package $1 from version $current_version to version $latest_version"
+  package=$1
+  cd $package
+  # echo "Checking $package..."
+
+  if jq -e "if (.dependencies.\"@chainlink/external-adapter-framework\" != null and .dependencies.\"@chainlink/external-adapter-framework\" != \"$LATEST_VERSION\") then true else false end" package.json > /dev/null ; then
+    current_version=$(jq -r '.dependencies."@chainlink/external-adapter-framework"' package.json) 
+    echo "Bumping dependency for package $package from $current_version to $LATEST_VERSION"
+    jq ".dependencies.\"@chainlink/external-adapter-framework\" = \"$LATEST_VERSION\"" package.json > package.json.tmp
+    mv package.json.tmp package.json
+    changed=true
+  fi
+
+  if jq -e "if (.devDependencies.\"@chainlink/external-adapter-framework\" != null and .devDependencies.\"@chainlink/external-adapter-framework\" != \"$LATEST_VERSION\") then true else false end" package.json > /dev/null ; then
+    current_version=$(jq -r '.devDependencies."@chainlink/external-adapter-framework"' package.json) 
+    echo "Bumping dev dependency for package $package from $current_version to $LATEST_VERSION"
+    jq ".devDependencies.\"@chainlink/external-adapter-framework\" = \"$LATEST_VERSION\"" package.json > package.json.tmp
+    mv package.json.tmp package.json
+    changed=true
+  fi
+
+  # Ignore scripts, that's not an adapter
+  if [[ $changed && $package != "scripts" ]]; then
```

Comment:
Wouldn't we want to include scripts? It does have it's own CHANGELOG so we could have a changeset for `@chainlink/ea-scripts`